test/valgrind/valgrindTestSuite.sh

```diff
@@ -352,6 +353,21 @@ function processResult()
 
 # -----------------------------------------------------------------------------
 #
+# Extract error info from valgrind output file
+#
+# It uses as argument the .out file to process
+#
+function memErrorInfo()
+{
+  filename=$1
+  # Get info from valgrind
+  memErrors=$(grep -n "Invalid write of size" $filename | wc -l)
```

Comment:
I really prefer to do an as exact match as possible. That way false positives are avoided as much as possible. If valgrind changes its syntax, we'll deal with it then.

Never seen an "invalid read", but that is probably easy to provoke. I'll try ...